src/content/en/updates/2018/10/animation-worklet.md

```diff
@@ -0,0 +1,493 @@
+before declaring the animation:
+
+    // index.html
+    await CSS.animationWorklet.addModule("passthrough-aw.js");
+    // ... WorkletAnimation initialization from above ...
+
+    // passthrough-aw.js
+    registerAnimator('passthrough', class {
+      animate(currentTime, effect) {
+        effect.localTime = currentTime;
+      }
+    });
+
+
+What is happening here? We are registering a class as an animator using the
+AnimationWorklet's `registerAnimator` call, giving it the name "passthrough".
+It's the same name we used in the WorkletAnimation constructor above. Once the
+registration is complete, the promise returned by `addModule` will resolve and
+we can start creating animations using that worklet.
+
+The `animate` method of our instance will be called for _every frame_ the
+browser wants to render, passing the `currentTime` of the animation's timeline
+as well as the effect that is currently being processed. We only have one
+effect, the `KeyframeEffect` and we are using `currentTime` to set the effect's
+`localTime`, hence why this animator is called "passthrough". With this code for
+the worklet, the WAAPI vs AnimationWorklet comparison above behave exactly the
+same, as you can see in the
+[demo](https://animation-worklet-article.glitch.me/index.html).
+
+## Master of time
+
+The `currentTime` parameter of our `animate` method is the `currentTime` of the
+timeline we passed to the `WorkletTimeline` constructor. In the previous
+example, we just passed that time through to the effect. But since this is
+JavaScript code, and we can _distort time_ 💫
+
+    function remap(minIn, maxIn, minOut, maxOut, v) {
+      return (v - minIn)/(maxIn - minIn) * (maxOut - minOut) + minOut;
+    }
+    registerAnimator('sin', class {
+      animate(currentTime, effect) {
+        effect.localTime =
+          remap(-1, 1, 0, 2000, Math.sin(currentTime * 2 * Math.PI / 2000));
+      }
+    });
+
+Note: `currentTime` _can_ be `NaN` in certain circumstances (more later). Your
+should keep that in mind when writing animation worklet code. Since all
+mathematical operations can handle `NaN` (they return `NaN` when one of their
+inputs is `NaN`) we are fine here!
+
+We are taking the `Math.sin` of the `currentTime`, and remapping that value to
+the range [0; 2000], which is the time range that our effect is defined for. Now
+[the animation looks very
+different](https://animation-worklet-article.glitch.me/sin.html), without having
+changed the keyframes or the animation's options. The worklet code can be
+arbitrarily complex, and allows you to programmatically define which effects are
+played in which order and to which extent.
+
+<video loop muted autoplay
+  src="https://storage.googleapis.com/webfundamentals-assets/animation-worklet/sin_x264.mp4">
+</video>
+
+## Options over Options
+
+You might find yourself wanting to reuse a worklet and just wanting to adjust
+some numbers. For this reason the WorkletAnimation constructor allows you pass
```

Comment:
This sentence can be shorter. 'You might want to reuse a worklet and change its numbers.'